daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -280,6 +280,43 @@ async function start() {
 }
 ```
 
+#### Bulk Subscribe to messages
+
+Dapr [supports bulk subscribing](https://v1-10.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/) a topic.
+
+- Bulk subscription through the `bulkSubscribeWithDefaultConfig` method, utilising default values for maxMessagesCount and maxAwaitDurationMs for related components.
+- Bulk subscription through the `bulkSubscribeWithConfig` method, where-in maxMessagesCount and maxAwaitDurationMs can be configured.
+
+```typescript
+import { DaprServer } from "@dapr/dapr";
+
+const pubSubName = "orderPubSub";
+const topic = "topicbulk";
+
+const DAPR_HOST = process.env.DAPR_HOST || "127.0.0.1";
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3502";
+const SERVER_HOST = process.env.SERVER_HOST || "127.0.0.1";
+const SERVER_PORT = process.env.APP_PORT || 5001;
+
+async function start() {
+  const server = new DaprServer(SERVER_HOST, SERVER_PORT, DAPR_HOST, DAPR_HTTP_PORT);
+
+  // Publish multiple messages to a topic with default config.
+  await client.pubsub.bulkSubscribeWithDefaultConfig(pubSubName, topic, (data) =>
+    console.log("Subscriber received: " + JSON.stringify(data)),
+  );
+
+  // Publish multiple messages to a topic with specific maxMessagesCount and maxAwaitDurationMs.
+  await client.pubsub.bulkSubscribeWithConfig(
+    pubSubName,
+    topic,
+    (data) => console.log("Subscriber received: " + JSON.stringify(data)),
+    100,
```

Comment:
Ref latest API spec changes, maybe make it in a parameters object with defaults?